### DISCUSSION TITLE
Elsa 3.1.3设计器中可视化Switch(flow)

### DISCUSSION THREAD

**Discussion by shxylgithub** [elsa-workflows/elsa-core#5693] [General]
Elsa 3.1.3设计器中可视化Switch(flow)

**Comment by glime-ai** [comment]
Could you please provide more details or clarify your question regarding the visual Switch (flow) in Elsa 3.1.3 designer? Are you asking how to use it, what features it has, or are you experiencing an issue with it? Any additional information would be helpful to provide you with an accurate answer.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=dRbvTb7IhCSYiI&utm_medium=github).
